src/components/ServiceQuestions.js

```diff
@@ -0,0 +1,245 @@
+        const titleInput = document.getElementById('titleInput');
+        const questionInput = document.getElementById('questionInput');
+        var title = titleInput.value;
+        var question = questionInput.value;
+        if (title === "" || question === "") {
+            alert("Please add a question")
+        }
+        this.serviceQuestionsService
+            .addServiceQuestion({title: title, question: question})
+            .then(() => this.refreshServiceQuestions());
+        titleInput.value = "";
+        questionInput.value = "";
+    }
+
+    updateServiceQuestion(id) {
+        const titleInput = document.getElementById('titleInput');
+        const questionInput = document.getElementById('questionInput');
+        var title = titleInput.value;
+        var question = questionInput.value;
+        this.serviceQuestionsService
+            .updateServiceQuestion({title: title, question: question}, id)
+            .then(() => this.refreshServiceQuestions());
+        titleInput.value = "";
+        questionInput.value = "";
+    }
+
+    deleteServiceQuestion(id) {
+        console.log('Delete ServiceQuestion Click happened');
+        this.serviceQuestionsService.deleteServiceQuestion(id)
+            .then(() => this.refreshServiceQuestions());
+    }
+
+    render() {
+        let div = (
+            <div>
+                <h3>Service Questions</h3>
+                <table className="table">
+                    <tr>
+                        <th>Title</th>
+                        <th>Question</th>
+                        <th> </th>
+                    </tr>
+                    <tr>
+                        <th>
+                            <input
+                                type="text"
+                                id="titleInput"
+                                placeholder="Title"
+                                title="Type in a name">
+                            </input>
+                        </th>
+                        <th>
+                            <input
+                                type="text"
+                                id="questionInput"
+                                placeholder="Question"
+                                title="Type in a name">
+                            </input>
+                        </th>
+                        <th>
+                            <button onClick={() => this.addServiceQuestion()}>Add</button>
+                        </th>
+                    </tr>
+                    <tbody>
+                    {
+                        this.state.serviceQuestions.slice(((this.state.pageNumber * this.state.itemsPerPage) - this.state.itemsPerPage), ((this.state.pageNumber * this.state.itemsPerPage) - 1))
+                            .map(serviceQuestion =>
+                                <tr key={serviceQuestion.id} onClick={this.props.history.push("/admin/service-questions/" + serviceQuestion.id)}>

```

Comment:
The OnClick for this was giving me issues. I had to get rid of it or else it kept rerouting to the details page on rendering.